src/main/java/com/google/api/codegen/config/GapicMethodConfig.java

```diff
@@ -148,17 +115,13 @@ static GapicMethodConfig createMethodConfig(
       batching =
           BatchingConfig.createBatching(
               diagCollector, methodConfigProto.getBatching(), methodModel);
-      if (batching == null) {
```

Comment:
Why are we removing this check? Do we want createBatching to be called and return null?

Reply:
If `createBatching()` returns null, it also adds an error to the `diagCollector`. I figured it was extraneous to have these null checks on everything, when we could just ask the `diagCollector` for # of errors logged at the end. 

tbf, this does require that any method that returns null inside the calling function to also log an error (which is true right now).

Reply:
OK sounds good - I thought we were now treating this situation as "not an error", but handling the error later SGTM